python/bench/repBench.py

```diff
@@ -75,7 +75,7 @@ def evaluate_unit(self, feature_1, feature_2, task):
             tcorr, corr_score, info = ellipse_overlap_H(
                 geo_info, feature_1, feature_2, option)
 
-            if corr_score.size == 0:
+            if len(corr_score) == 0 or corr_score.size == 0:
```

Comment:
If no overlap found then corr score will be empty list, and thus throw an error with `corr_score.size`. Checks list attribute first.